docs/index.md

```diff
@@ -48,6 +48,20 @@ pip install git+https://github.com/tableau/server-client-python.git@development
 Note that the version from the development branch should not be used for production code. The methods and endpoints in the
 development version are subject to change at any time before the next stable release.
 
+### Install on an offline machine
+
+To install TSC onto a machine without internet connection, use the following steps:
+
+1) Download the [setup package](https://pypi.org/project/tableauserverclient/#files){:target="_blank"}.
+2) Manually install the dependent Python libraries.
+
+   > **Note:**  The way python dependencies are configured in the TSC set up package requires either for them to be preinstalled when installing TSC, or for network to be available to retrieve them. Offline installation currently requires manual installation of the following python libraries (and their dependencies) before intalling TSC:
```

Comment:
I can put my MR out to remove the need for the test/test runner, so this just lists requests.

And maybe a line about "needing to download the dependencies for requests`?